### ISSUE TITLE
🐞 [Bug]: Broken Link tfrobot readme and others

### ISSUE THREAD

**Issue by mik-tf** [threefoldtech/tfgrid-sdk-go#844]
### What happened?

- There is a broken link on the [tfrobot readme](https://github.com/threefoldtech/tfgrid-sdk-go/blob/development/tfrobot/README.md)
- We can either remove this sentence with the link, or add an example. In either case, it'd be good to have both json and yaml.
- [Line 54](https://github.com/threefoldtech/tfgrid-sdk-go/blob/8d88cb1f97b59f6c2a8294735d093dcfbd897f6f/tfrobot/README.md?plain=1#L54)

### Relevant log output

```console
404 - page not found
The development branch of tfgrid-sdk-go does not contain the path tfrobot/example/conf.yaml.
```

**Comment by xmonader** [comment]
Thanks, should be fixed here fed0436b0a7a158d15cc3d1ee4b0e5a0bb58c7ce

**Comment by AbdelrahmanElawady** [comment]
Verified:

Link has been removed and example in README updated.
